src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PackageReference.xaml

```diff
@@ -56,13 +56,13 @@
                     Visible="False" 
                     ReadOnly="True" />
 
-    <StringListProperty Name="Dependencies"
+    <!--<StringListProperty Name="Dependencies"
```

Comment:
@basoundr has more context on this, as he was seeing errors with this rule present. This Dependencies rule should only be in the ResolvedPackageReference.xaml rule file because that is where dependencies apply. I commented it out here, but I will go ahead and delete it.